doc/075_scripting.rst

```diff
@@ -14,8 +14,99 @@
 Scripting
 #########################
 
-This is a list of how certain tasks may be accomplished when you use
-restic via scripts.
+This section provides an overview of available environment variables and how
+certain tasks may be accomplished when you use restic via scripts.
+
+.. _environment-variables:
+
+Environment Variables
+*********************
+
+In addition to command-line options, restic supports passing various options in
+environment variables. The following lists these environment variables:
+
+.. code-block:: console
+
+    RESTIC_REPOSITORY_FILE              Name of file containing the repository location (replaces --repository-file)
```

Comment:
This is not a new issue, but I think this is slightly misleading: the option overrides the environment variable when used, but "replaces" suggests it is the other way around.

It is hard to express this compactly, though.  Perhaps "(alternative to --repository-file)" or "(option --repository-file)".  Another option is to describe behaviour explicitly in a paragraph:

> When both an environment variable and the option it replaces is given, the command line option has precedence.

With the latter paragraph, the texts for each environment variable does not need changing.

Reply:
I do not understand why you want to make the documentation harder to use, but if you insist.

Reply:
I'm not looking to make it harder. I am looking for replies explaining the use cases, hence my questions. Can you answer them?